projects/plugins/jetpack/changelog/change-response-export-field-names

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+Improve field/column name handling when exporting responses for proper "Date", "IP Address" and other column names.
```

Comment:
I'm thinking it may be worth being more detailed in our changelog entry here. I can imagine some folks may have workflows set up around the export files, so we would want to make it clear that we are making changes there.

Maybe something like:

```suggestion
Contact Form: update the name of the columns used when exporting Contact Form responses to CSV. The file will now use clear and detailed column names.
```